---
title: Publication
permalink: /Publication/
---

## 2025

_Uncovering the Mechanistic Landscape of Regulatory DNA with Deep Learning_<br>
<b>E Seitz</b>, <b>DM McCandlish</b>, <b>JB Kinney</b>, <b>PK Koo</b><br>
bioRxiv ([Preprint](https://www.biorxiv.org/content/10.1101/2025.10.07.681052v1.full), [Code](https://github.com/evanseitz/seam-nn))<br> 

_Evaluating the representational power of pre-trained DNA language models for regulatory genomics_<br>
<b>Z Tang</b>, <b>N Somia</b>, <b>Y Kang</b>, <b>PK Koo</b><br>
bioRxiv ([Preprint](https://www.biorxiv.org/content/10.1101/2024.02.29.582810v1), [Code](https://github.com/amberT15/LLM_eval)) <br> 
_Presented at the 2023 ICML Workshop on Computational Biology_ 
_Selected for oral presentation at 2024 CSHL Meeting on Systems Biology_ 
_[Research Highlight at Nature Methods](https://www.nature.com/articles/s41592-025-02829-6)_

_Towards interpretable prediction of recurrence risk in breast cancer using pathology foundation models_<br>
<b>JR Kaczmarzyk</b>, S Van Alsten, AJ Cozzo, R Gupta, <b>PK Koo</b>, MA Troester, KA Hoadley, JH Saltz<br>
arXiv ([Preprint](https://arxiv.org/abs/2508.12025))<br>

_Machine Learning for Protein Science and Engineering_<br>
<b>PK Koo</b>, C Dallago, A Nambiar, KK Yang<br>
Cold Spring Harbor Perspectives in Biology ([Book](https://cshperspectives.cshlp.org/content/early/2025/05/03/cshperspect.a041877.abstract))<br>

_Massive experimental quantification allows interpretable deep learning of protein aggregation_<br>
M Thompson, M Mart√≠n, TS Olmo, <b>C Rajesh</b>, <b>PK Koo</b>, B Bolognesi, B Lehner<br>
Science Advances ([Paper](https://www.science.org/doi/full/10.1126/sciadv.adt5111))<br>

_Reusable specimen-level inference in computational pathologys_<br>
<b>JR Kaczmarzyk</b>, R Sharma, <b>PK Koo</b>, JH Saltz<br>
arXiv ([Preprint](https://arxiv.org/abs/2501.05945), [Code](https://github.com/SBU-BMI/SpinPath))<br>

_Probabilistic and machine-learning methods for predicting local rates of transcription elongation from nascent RNA sequencing data_
L Liu, Y Zhao, R Hassett, <b>S Toneyan</b>, <b>PK Koo</b>, A Siepel
Nucleic Acids Research ([Paper](https://academic.oup.com/nar/article/53/4/gkaf092/8020229))<br>


## 2024

_Uncertainty-aware genomic deep learning with knowledge distillation_<br>
<b>JL Zhou</b>, <b>K Rizzo</b>, <b>Z Tang</b>, <b>PK Koo</b><br>
bioRxiv ([Paper](https://www.biorxiv.org/content/10.1101/2024.11.13.623485), [Code](https://github.com/zrcjessica/ensemble_distillation))<br>


_Explainable AI for computational pathology identifies model limitations and tissue biomarkers_<br>
<b>JR Kaczmarzyk</b>, JH Saltz, <b>PK Koo</b><br>
arXiv ([Preprint](https://arxiv.org/abs/2409.03080), [Code](https://github.com/kaczmarj/HIPPO))<br>


_Designing DNA With Tunable Regulatory Activity Using Discrete Diffusion_<br>
<b>A Sarkar</b>, <b>Z Tang</b>, <b>C Zhao</b>, <b>PK Koo</b><br>
bioRxiv ([Preprint](https://www.biorxiv.org/content/10.1101/2024.05.23.595630v1.abstract), [Code](https://anonymous.4open.science/r/D3-DNA-Discrete-Diffusion-3E42/))<br> 
 Selected for oral presentation at 2024 MLCB


_Analysis of single-cell CRISPR perturbations indicates that enhancers predominantly act multiplicatively_<br>
JL Zhou, K Guruvayurappan, <b>S Toneyan</b>, HV Chen, AR Chen, <b>PK Koo</b>, G McVicker<br>
Cell Genomics ([Paper](https://www.cell.com/cell-genomics/fulltext/S2666-979X(24)00291-X), [Code](https://github.com/mcvickerlab/GLiMMIRS))<br>


_Interpreting Cis-Regulatory Interactions from Large-Scale Deep Neural Networks for Genomics_<br>
<b>S Toneyan</b>, <b>PK Koo</b><br>
Nature Genetics ([Article](https://www.nature.com/articles/s41588-024-01923-3), [Code](https://github.com/p-koo/creme-nn)) <br>
_Selected for oral presentation at 2023 MLCB_


_Interpreting Cis-Regulatory Mechanisms from Genomic Deep Neural Networks using Surrogate Models_<br>
<b>E Seitz</b>, <b>DM McCandlish</b>, <b>JB Kinney</b>, <b>PK Koo</b><br>
Nature Machine Intelligence ([Article](https://www.nature.com/articles/s42256-024-00851-5), [Code](https://github.com/evanseitz/squid-nn))<br> 


_EvoAug-TF: Extending evolution-inspired data augmentations for genomic deep learning to TensorFlow_<br>
<b>Y Yu</b>, <b>S Muthukumar</b>, <b>PK Koo</b><br>
Bioinformatics ([Article](https://academic.oup.com/bioinformatics/article/40/3/btae092/7609674), [Code](https://github.com/p-koo/evoaug-tf))<br> 



## 2023


_Revisiting initialization strategies for deep learning in regulatory genomics_<br> 
<b>C Rajesh</b>, <b>Z Tang</b>, <b>PK Koo</b><br>
2023 Machine Learning in Computational Biology


_Building foundation models for regulatory genomics requires rethinking large language models_<br> 
<b>Z Tang</b>, <b>PK Koo</b><br>
ICML 2023 Workshop on Computational Biology ([Workshop paper](https://icml-compbio.github.io/2023/papers/WCBICML2023_paper92.pdf))


_EvoAug: improving generalization and interpretability of genomic deep neural networks with evolution-inspired data augmentations_<br>
<b>NK Lee</b>, <b>S Toneyan</b>, <b>Z Tang</b>, <b>PK Koo</b><br>
Genome Biology ([Article](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02941-w), [Code](https://github.com/p-koo/evoaug))<br> 


_Correcting gradient-based interpretations of deep neural networks for genomics_<br>
<b>A Majdandzic</b>, <b>C Rajesh</b>, <b>PK Koo</b><br>
Genome Biology ([Article](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02956-3), [Code](https://github.com/p-koo/GradientCorrection)) <br>


_ChampKit: A framework for rapid evaluation of deep neural networks for patch-based histopathology classification_<br>
<b>JR Kaczmarzyk</b>, R Gupta, TM Kurc, S Abousamra, JH Saltz, <b>PK Koo</b><br>
Computer Methods and Programs in Biomedicine ([Article](https://www.sciencedirect.com/science/article/pii/S0169260723002961), [Code](https://github.com/kaczmarj/champkit))<br>


_ETV6 dependency in Ewing sarcoma by antagonism of EWS-FLI1-mediated enhancer activation_<br>
Y Gao, XY He, ..., <b>S Toneyan</b>, ..., <b>PK Koo</b>, ..., C Vakoc<br>
Nature Cell Biology ([Article](https://www.nature.com/articles/s41556-022-01060-1))<br>


_ResidualBind: Uncovering Sequence-Structure Preferences of RNA-Binding Proteins with Deep Neural Networks_<br>
<b>PK Koo</b>, M Ploenzke, P Anand, S Paul, <b>A Majdandzic</b> <br>
RNA Structure Prediction, Methods in Molecular Biology ([Chapter](https://link.springer.com/protocol/10.1007/978-1-0716-2768-6_12))<br>



## 2022


_Selecting deep neural networks that yield consistent attribution-based interpretations for genomics_<br>
<b>A Majdandzic</b>, <b>C Rajesh</b>, <b>Z Tang</b>, <b>S Toneyan</b>, <b>E Labelson</b>, <b>R Tripathy</b>, <b>PK Koo</b><br>
2022 Machine Learning in Computational Biology ([Proceedings](https://proceedings.mlr.press/v200/majdandzic22a.html), [Code](https://github.com/crajesh6/acme))<br>
_Selected for oral presentation at MLCB_

_Learning single-cell chromatin accessibility profiles using meta-analytic marker genes_<br>
RK Kawaguchi, <b>Z Tang</b>, S Fischer, <b>Chandana Rajesh</b>, <b>R Tripathy</b>, <b>PK Koo</b>, J Gillis <br>
Briefings in Bioinformatics ([Article](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac541/6957249?login=false), [Code](https://github.com/carushi/Catactor))<br>

_Evaluating deep learning for predicting epigenomic profiles_<br>
<b>S Toneyan</b>, <b>Z Tang</b>, <b>PK Koo</b><br>
Nature Machine Intelligence ([Article](https://www.nature.com/articles/s42256-022-00570-9), [Code](https://github.com/shtoneyan/gopher))<br>

_Interpreting Potts and Transformer Protein Models Through the Lens of Simplified Attention_<br>
N Bhattacharya, N Thomas, R Rao, J Dauparas, <b>PK Koo</b>, D Baker, YS Song, S Ovchinnikov <br>
Pacific Symposium on Biocomputing ([Article](https://www.worldscientific.com/doi/abs/10.1142/9789811250477_0004), [Code](https://github.com/nickbhat/mogwai.git))


## 2021
_Uncovering motif interactions from convolutional-attention networks for genomics_<br>
<b>R Ghotra</b>, <b>N Lee</b>, <b>PK Koo</b> <br>
NeurIPS 2021 AI for Science Workshop ([Workshop paper](https://openreview.net/forum?id=ITOQhccyRsk&referrer=%5BAuthor%20Console%5D(%2Fgroup%3Fid%3DNeurIPS.cc%2F2021%2FWorkshop%2FAI4Science%2FAuthors%23your-submissions)), [Code]())   <br>
_Selected for oral presentation_ -- <b>Best Paper Award</b>


_Towards trustworthy explanations with gradient-based attribution methods_<br>
<b>EL Labelson</b>, <b>R Tripathy</b>, <b>PK Koo</b> <br>
NeurIPS 2021 AI for Science Workshop ([Workshop paper](https://openreview.net/forum?id=LGgo0wPM2MF&referrer=%5BAuthor%20Console%5D(%2Fgroup%3Fid%3DNeurIPS.cc%2F2021%2FWorkshop%2FAI4Science%2FAuthors%23your-submissions)), [Code]())  


_End-to-end learning of multiple sequence alignmentswith differentiable Smith-Waterman_<br>
S Petty, N Bhattacharya, R Rao, J Dauparas, N Thomas, J Zhou, AM Rush, <b>PK Koo</b>, S Ovchinnikov <br>
NeurIPS 2021 Machine Learning for Structural Biology Workshop ([Workshop paper](https://www.biorxiv.org/content/10.1101/2021.10.23.465204v1), [Code](https://github.com/spetti/SMURF))  <br>
_Selected for oral presentation_


_Statistical correction of input gradients for black box models trained with categorical input features_<br>
<b>A Majdandzic</b>, <b>PK Koo</b> <br>
ICML 2021 Workshop on Computational Biology ([Workshop paper](https://icml-compbio.github.io/2021/papers/WCBICML2021_paper_56.pdf), [Code](https://github.com/p-koo/gradient_correction))   <br>
_Selected for spotlight presentation_


_Designing Interpretable Convolution-Based Hybrid Networks for Genomics_<br>
<b>R Ghotra</b>, <b>NK Lee</b>, <b>R Tripathy</b>, <b>PK Koo</b> <br>
ICML 2021 Workshop on Computational Biology ([Workshop paper](https://icml-compbio.github.io/2021/papers/WCBICML2021_paper_28.pdf), [Code](https://github.com/p-koo/hybrid_genomics)) 

_Representation learning of genomic sequence motifs via information maximization_<br>
<b>NK Lee</b>, <b>PK Koo</b> <br>
ICML 2021 Workshop on Computational Biology ([Workshop paper](https://icml-compbio.github.io/2021/papers/WCBICML2021_paper_58.pdf)) 

_Global Importance Analysis: A Method to Quantify Importance of Genomic Features in Deep Neural Networks_<br>
<b>PK Koo</b>, <b>A Majdandzic</b>, M Ploenzke, P Anand, S Paul <br>
PLOS Computational Biology ([Article](https://www.biorxiv.org/content/10.1101/2020.09.08.288068v1), [Code](https://github.com/p-koo/residualbind))

_Improving Representations of Genomic Sequence Motifs in Convolutional Networks with Exponential Activations_<br>
<b>PK Koo</b>, M Ploenzke <br>
Nature Machine Intelligence ([Article](https://www.nature.com/articles/s42256-020-00291-x), [Code](https://github.com/p-koo/exponential_activations))

_Exploiting marker genes for robust classification and characterization of single-cell chromatin accessibility_<br>
RK Kawaguchi, <b>Z Tang</b>, S Fischer, <b>R Tripathy</b>, <b>PK Koo</b>, J Gillis <br>
bioRxiv ([Preprint](https://www.biorxiv.org/content/10.1101/2021.04.01.438068v1), [Code](https://github.com/carushi/Catactor))

_Single Layers of Attention Suffice to Predict Protein Contacts_<br>
N Bhattacharya, N Thomas, R Rao, J Dauparas, <b>PK Koo</b>, D Baker, YS Song, S Ovchinnikov <br>
ICLR 2021 Workshop on Energy Based Models ([Workshop paper](https://www.biorxiv.org/content/10.1101/2020.12.21.423882v2), [Code](https://github.com/nickbhat/mogwai.git))


## 2020

_Deep Learning for Inferring Transcription Factor Binding Sites_<br>
<b>PK Koo</b>, M Ploenzke <br>
Current Opinion in Systems Biology ([Review](https://www.sciencedirect.com/science/article/pii/S2452310020300032?via%3Dihub))

_ZBED2 is an antagonist of interferon regulatory factor1 and modifies cell identity in pancreatic cancer_<br>
TD Somerville, Y Xu, XS Wu, D Maia-Silva, SK Hur, LMN de Almeida, JB Preall, <b>PK Koo</b>, CR Vakoc <br>
Proceedings of the National Academy of Sciences ([Article](https://www.pnas.org/content/117/21/11471.short))


_N-WASP Regulates the Mobility of the B Cell Receptor and Co-Receptors During Signaling Activation_<br>
I Rey-Suarez, B Wheatley, <b>PK Koo</b>, Z Shu, S Mochrie, W Song, H Shroff, A Upadhyaya <br>
Nature Communications ([Article](https://www.nature.com/articles/s41467-020-14335-8))


_The Structure-Fitness Landscape of Pairwise Relations in Generative Sequence Models_<br>
D Marshall, H Wang, M Stiffler, J Dauparas, <b>PK Koo</b>, S Ovchinnikov <br>
NeurIPS 2020 Workshop on Machine Learning for Structural Biology ([Workshop paper](https://www.biorxiv.org/content/10.1101/2020.11.29.402875v1), [Code](https://github.com/sokrypton/seqsal_v2))



## 2019


_Representation Learning of Genomic Sequence Motifs with Convolutional Neural Networks_<br>
<b>PK Koo</b>, SR Eddy <br>
PLOS Computational Biology ([Article](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007560), [Code](https://github.com/p-koo/learning_sequence_motifs))


_Interpreting Deep Neural Networks Beyond Local Attribution Methods: Quantifying Global Importance of Features_<br>
<b>PK Koo</b>\*, M Ploenzke\* <br>
Machine Learning in Computational Biology Meeting, 2019 ([Extended abstract](https://www.biorxiv.org/content/10.1101/2020.02.19.956896v1.abstract))


_Improving Convolutional Network Interpretability with Exponential Activations_<br>
<b>PK Koo</b>\*, M Ploenzke\* <br>
ICML 2019 Workshop for Computational Biology ([Workshop paper](https://www.biorxiv.org/content/10.1101/650804v1.abstract), [Code](https://github.com/p-koo/cnn_exponential_activations))


_Robust Neural Networks are More Interpretable for Genomics_<br>
<b>PK Koo</b>, S Qian, G Kaplun, V Volf, D Kalimeris <br>
ICML 2019 Workshop for Computational Biology  ([Workshop paper](https://www.biorxiv.org/content/10.1101/657437v1.abstract), [Code](https://github.com/p-koo/uncovering_regulatory_codes))

_Unified framework for modeling multivariate distributions in biological sequences_<br>
J Dauparas, H Wang, A Swartz, <b>PK Koo</b>, M Nitzan, S Ovchinnikov <br>
ICML 2019 Workshop for Computational Biology ([Workshop paper](https://arxiv.org/abs/1906.02598), [Code](https://github.com/sokrypton/seqmodels))

_A Demonstration of Unsupervised Machine Learning in Species Delimitation_<br>
S Derkarabetian, S Castillo, <b>PK Koo</b>, S Ovchinnikov, M Hedin <br>
Molecular Phylogenetics and Evolution 139, 106562 ([Article](https://www.sciencedirect.com/science/article/pii/S1055790319301721))



<hr>

## 2018

_Inferring Sequence-Structure Preferences of RNA-Binding Proteins with Convolutional Residual Networks_<br>
<b>PK Koo</b>, P Anand, SB Paul, SR Eddy <br>
bioRxiv, 418459 ([Preprint](https://www.biorxiv.org/content/10.1101/418459v1.abstract))

_Representation Learning of Genomic Sequence Motifs with Convolutional Neural Networks_<br>
<b>PK Koo</b>, SR Eddy <br>
bioRxiv, 362756 ([Preprint](https://www.biorxiv.org/content/10.1101/362756v4), [Code](https://github.com/p-koo/learning_sequence_motifs))

_Applying Perturbation Expectation-Maximization to Protein Trajectories of Rho GTPases_<br>
<b>PK Koo</b>, SR Eddy <br>
Rho GTPases, 57-70 ([Chapter](https://link.springer.com/protocol/10.1007/978-1-4939-8612-5_5), [Code](https://github.com/p-koo/pEMv2))


<hr>

## 2017


_Inferring Functional Neural Connectivity With Deep Residual Convolutional Networks_<br>
TW Dunn, <b>PK Koo</b> <br>
bioRxiv, 141010 ([Preprint](https://www.biorxiv.org/content/10.1101/141010v2.abstract), [Code](https://github.com/spoonsso/TFconnect))



<hr>

## 2016

_Systems-level Approach to Uncovering Diffusive States and Their Transitions from Single-Particle Trajectories_<br>
<b>PK Koo</b>, SGJ Mochrie <br>
Phsical Review E 94 (5), 052412 ([Article](https://journals.aps.org/pre/abstract/10.1103/PhysRevE.94.052412), [Code](https://github.com/p-koo/pEMv2))

_Improved Determination of Subnuclear Position Enabled by Three-Dimensional Membrane Reconstruction_<br>
Y Zhao, SM Schreiner, <b>PK Koo</b>, P Colombi, MC King, SGJ Mochrie <br>
Biophysical Journal 111 (1), 19-24 ([Article](https://www.sciencedirect.com/science/article/pii/S0006349516303617), [Code](https://github.com/mochrielab/3DMembraneReconstruction))



<hr>

## 2015


_The Tethering of Chromatin to the Nuclear Envelope Supports Nuclear Mechanics_<br>
SM Schreiner\*, <b>PK Koo</b>\*, Y Zhao, SGJ Mochrie, MC King <br>
Nature Communications 6, 7159 ([Article](https://www.nature.com/articles/ncomms8159))


_Extracting Diffusive States of Rho GTPase in Live Cells: Towards In Vivo Biochemistry_<br>
<b>PK Koo</b>\*, M Weitzman\*, CR Sabanaygam, KL van Golen, SGJ Mochrie <br>
PLOS Comptuational Biology 11 (10), e1004297 ([Article](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004297), [Code](https://github.com/p-koo/pEM))
<a href="https://f1000.com/prime/725888939"><img width="100" src="{{site.baseurl}}/images/F1000Prime.png"></a>
<br>
\* Shared co-first author
<br>

For a full list, click <a href="https://scholar.google.com/citations?user=zoAsQGwAAAAJ&hl=en">here</a>.
<br>
<br>
<br>
<br>


&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;
&nbsp;


























